src/Microsoft.AspNetCore.SystemWebAdapters/Utilities/StringUtil.cs

```diff
@@ -0,0 +1,15 @@
+namespace System.Web.Util;
```

Comment:
nit: these files need the header. Apparently we don't have a code fix for that :(

Reply:
Ah - Yeah - should probably have put the original source mention like I did in VirtualPathUtility.cs.
I did tweak `StringUtl.StringStartsWithIgnoreCase` to use the newer StartsWith overload (I assume it was written against a much earlier version of .Net and never changed). I did do a quick BenchmarkDotNet test of the original code vs my version and got a x2 performance improvement. It still needs the wrapper due to different exception handing for null values.

Reply:
Added attribution for the classes, and the header as per the other files.